internal/common/echo_logrus.go

```diff
@@ -4,11 +4,18 @@ import (
 	"context"
 	"encoding/json"
 	"io"
+	"runtime"
 
 	"github.com/labstack/gommon/log"
 	"github.com/sirupsen/logrus"
 )
 
+type ctxKey int
+
+const (
+	LoggingFrameCtx ctxKey = iota
+)
```

Comment:
It is a good practice not to export context key types and values so other packages cannot modify the values. Here it is a mix - you export the value but not the type.

Reply:
So we should change this in `cmd/image-builder/logging.go` too? (where I got the pattern from)